nubis/puppet/files/nubis-taint-reap

```diff
@@ -25,7 +26,7 @@ if [ "$TAINTED" -gt "1" ]; then
 
   if [ "$EXPIRE_ENABLED" == "1" ]; then
     echo "Expiring this instance NOW"
-    echo "NOTIMPLEMENTEDYET: shutdown -p now"
+    echo "NOTIMPLEMENTEDYET: shutdown -h now"
```

Comment:
Another one that will be more portable across distro flavors, and we don't actually do it yet!